Model/Admin/Source/GetStoreNames.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Wizzy\Search\Model\Admin\Source;
+
+use Magento\Store\Model\StoreManagerInterface;
+
+class GetStoreNames
+{
+    private $storeManager;
+ 
+    public function __construct(StoreManagerInterface $storeManager)
+    {
+        $this->storeManager = $storeManager;
+    }
+
+    public function toOptionArray()
+    {
+        $storeNames = [
+            [
+                "value" => '',
+                "label" => 'select'
+            ]
+        ];
+
+        foreach ($this->storeManager->getStores() as $store) {
```

Comment:
We already have this logic inside a Store Manager. Which we use to fetch list of stores in products indexer. Please use the same.